format/spec.md

```diff
@@ -786,9 +786,9 @@ Notes:
 
 #### First Row ID Assignment
 
-When adding a new data manifest file, its `first_row_id` field is assigned the value of the snapshot's `first_row_id` plus the sum of `added_rows_count` for all data manifests that preceded the manifest in the manifest list.
+When adding a new data manifest file, its `first_row_id` field is assigned the value of the snapshot's `first_row_id` plus the sum of `added_rows_count` and `existing_rows_count` for all data manifests that preceded the manifest in the manifest list and were assigned a `first_row_id`.
```

Comment:
About you said "had a null ..." and i think that's a little bit clearer than "were assigned a"

Reply:
Updated, but this is a bit tricky because these are simply new manifests. Having a null `first_row_id` is an implementation detail that is not written into metadata. I've updated it to say both:

> When adding a new data manifest file, its `first_row_id` field is assigned the value of the snapshot's `first_row_id` plus the sum of `added_rows_count` and `existing_rows_count` for all new data manifests that preceded it in the manifest list; that is, those that had a null `first_row_id` and were assigned one.

I also reforamtted this secitn a little.